govcd/vm_test.go

```diff
@@ -2021,6 +2174,37 @@ func (vcd *TestVCD) Test_VMChangeMemory(check *C) {
 }
 
 func (vcd *TestVCD) Test_AddRawVm(check *C) {
+	vapp, vm := createNsxtVAppAndVm(vcd, check)
+
+	// Check that vApp did not lose its state
```

Comment:
Before using vApp and VM, we should check that they are not nil

Reply:
Good catch! Fixed